ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -541,6 +608,9 @@ open class BottomCommandingController: UIViewController {
 
     private var bottomBarHidingAnimator: UIViewPropertyAnimator?
 
+    // Constraints attaching self.layoutGuide to the current commanding surface (bar or a sheet)
+    private var layoutGuideConstraints: [NSLayoutConstraint] = [NSLayoutConstraint]()
```

Comment:
Using this syntax is a bit duplicate with providing the type hint. Could just be = []?

Reply:
I think the linter will complain but I will try :D

Reply:
Linter is smarter than I thought. Done!